client/my-sites/site-settings/main.jsx

```diff
@@ -25,31 +24,21 @@ import SiteSettingsNavigation from './navigation';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import JetpackDevModeNotice from './jetpack-dev-mode-notice';
 
-/**
- * Module vars
- */
-const debug = debugFactory( 'calypso:my-sites:site-settings' );
-
 export class SiteSettingsComponent extends Component {
-	constructor( props ) {
-		super( props );
-
-		// bound methods
-		this.updateSite = this.updateSite.bind( this );
-
-		this.state = {
-			site: this.props.sites.getSelectedSite()
-		};
-	}
 
```

Comment:
Not sure why our linter does not yell about trailing empty lines in classes. Care to remove this one?